### ISSUE TITLE
document with mistake in ppc64le installtion pages.

### ISSUE THREAD

**Issue by titawork** [omnisci/omniscidb#356]
When I install  Mapd in a ppc64le mechine with tarball. 
 in this page
https://www.omnisci.com/platform/downloads/OS-installation-recipes?compute_mode=gpucpu&environment=linuxubuntu&architecture=ppc64le&install_mode=tarball

the link of the tar file is  
https://releases.omnisci.com/os/tar/omnisci-os-latest-Linux-x86_64.tar.gz
It must be wrong. The x86_64.tar.gz can't install in a ppc64le mechine.
Is the link wrong? or there is no tar file for ppc64le ?

**Comment by randyzwitch** [comment]
Thanks for your patience @titawork. I spoke with our DevOps team, this is a mistake in our documentation in that we don't currently build the OSS version for ppc64le (we just build Enterprise Edition).

Is your interest a one-off type of use or would you be interested in having the OSS build for PowerPC going forward? The short-term solution would be to compile it for now, but I can request that the OSS version gets built on PowerPC as well if that's of interest.

**Comment by titawork** [comment]
Thank you very much for your reply. 
If you have a MapD on the PPC64le platform with open source installation, it will be great and save、. Envn We also plan to build PPCLE64 version ourselves，   We are working with IBM to promote the high-performance PPC64Le platform machine, and MapD will be a very important part. We are now exploring the application of MapD to various clearing scenarios.

**Comment by titawork** [comment]
> Thanks for your patience @titawork. I spoke with our DevOps team, this is a mistake in our documentation in that we don't currently build the OSS version for ppc64le (we just build Enterprise Edition).
> 
> Is your interest a one-off type of use or would you be interested in having the OSS build for PowerPC going forward? The short-term solution would be to compile it for now, but I can request that the OSS version gets built on PowerPC as well if that's of interest.

One more question. Do MapD 4.4.2  for PPC64le support NVIDIA Tesla T4 or not ? 
When I test on Redhat 7.4 with such MapD and GPU, it seems GPUs usage utilization rate is always 0%, but CPUs is 100%, regardless run in CPU mode or GPU mode,

**Comment by randyzwitch** [comment]
I don't have a Tesla T4 system to test against, but offhand I can't think of any reason it wouldn't work. What version of CUDA and NVIDIA drivers are you using? @andrewseidl , is there anything else we should check to troubleshoot this?

**Comment by cdessanti** [comment]
> One more question. Do MapD 4.4.2 for PPC64le support NVIDIA Tesla T4 or not ?
> When I test on Redhat 7.4 with such MapD and GPU, it seems GPUs usage utilization rate is always 0%, but CPUs is 100%, regardless run in CPU mode or GPU mode,

I run MapD 4.4.2 x86 version on Turing Hardware (RTX 2080ti) without any problem. You should give us logs of mapd_server to help us to figure out the problem you are facing

**Comment by titawork** [comment]
> I run MapD 4.4.2 x86 version on Turing Hardware (RTX 2080ti) without any problem. You should give us logs of mapd_server to help us to figure out the problem you are facing

Thanks! I tried several  times, MapD can work on NVIDIA Tesla T4. I think the problem should be my PPC64LE machine run slower than my X86_64 machine.

**Comment by andrewseidl** [comment]
Hi @titawork, do other CUDA programs work on your ppc64le + T4 machine? Even the CUDA samples fail for us on our Talos II box with Pascal GPUs, though I've heard rumors that Turing may work. Most programs I've tried will fail when initializing their first context / performing their first `cuda_malloc`. Before hearing the Turing rumors I suspected it was an issue with the drivers always assuming that they're using SXM2 on ppc64le, never PCIe.

If it is this issue, the `mapd_server` / `omnisci_server` logs (under `mapd_log` in your `data` directory) should have a line saying something like "No GPUs detected, falling back to CPU mode."

If it helps, here's an unofficial build of OmniSciDB 4.7.0 for ppc64le: https://mapd-core-os-builds.s3.amazonaws.com/omnisci-os-4.7.0-20190626-4559864637-Linux-ppc64le.tar.gz

**Comment by titawork** [comment]
Thank you very much for your newest omnisci version! 
Other programs works right on my  ppc64le + T4 machine, and also omnisci. 
I think the problem is I set one core with 4 Hyper-Threading on my ppc64le machine. So each Hyper-Threading decrease in ability. As omnisci use all logic cores for tasks, we know that more threads means more competition. After set one core with 2 Hyper-Threading， we got good performance.
